app/authorization/server/startup.js

```diff
@@ -134,7 +132,7 @@ Meteor.startup(function() {
 	];
 
 	for (const role of defaultRoles) {
-		Roles.upsert({ _id: role.name }, { $setOnInsert: { scope: role.scope, description: role.description || '', protected: true, mandatory2fa: false } });
+		Roles.createOrUpdate(role.name, role.scope, role.description, true, false);
```

Comment:
Because the Permissions are changed by the user, so we only create if it doesn't exist, but the role can't be changed so we can update without a migration when changing the scope or description.